acquisitions/src/main/resources/thunderjet/consortia/tenant-utils/tenant.feature

```diff
@@ -0,0 +1,654 @@
+Feature: Tenant object in mod-consortia api tests
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+    * configure retry = { count: 10, interval: 15000 }
+
+  @Positive
+  Scenario: Create users with all different types to verify the case of skipping some types when enabling tenant
```

Comment:
I believe we can delete extra scenarios here to keep only creation of 3 tenant. Start with creation of this users

Reply:
I believe this is done is last commit